test/compilable/extra-files/header18365.d

```diff
@@ -0,0 +1,6 @@
+module foo.bar.ba;
+nothrow pure @nogc @safe package(foo) 
+{
+	void foo();
+	nothrow pure @nogc @safe package(foo.bar) void foo2();
+}
```

Comment:
why is this necessary? You can add your test cases in the compilable/extra-files/header1.d or header2.d or header3.d.
There is no need to add an extra file.

Reply:
Because of `module foo.bar.ba;`